storage-access.bs

```diff
@@ -302,6 +271,30 @@ To the [=parse a sandboxing directive=] algorithm, add the following under step
 <li>The [=sandbox storage access by user activation flag=], unless <var ignore>tokens</var> contains the <dfn export attr-value for=iframe/sandbox>allow-storage-access-by-user-activation</dfn> keyword.
 </ul>
 
+<h2 id="permissions-integration">Permissions Integration</h2>
+
+The Storage Access API defines a [=powerful feature=] identified by the [=powerful feature/name=] `"storage-access"`.
```

Comment:
It would be nice to add a `permission` type and export the "storage-access" token: see https://tabatkins.github.io/bikeshed/#dfn-types

(so something like `<dfn export permission>storage-access</dfn>`, and later reference w/ `<a permission>storage-access>`, modulo however you want to handle quotes and `<code>`.)

Reply:
Oh, that's a thing I can already use today? Neat.

Reply:
Updated, thanks for the tip!